rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -502,25 +502,22 @@ upgrade_database() {
   # Now check the fileFormat in the eventlog
   upgrade_eventlog
 
-  # - All versions: Check for the PostgreSQL version (>= 8.4)
+  # - All versions: Check for the PostgreSQL version
   STEP="Check for the current PosgresSQL version"
 
-  # If installing on SLES11 and PostgreSQL 8.3 was used
+  # If coming from an old SLES11 install
```

Comment:
technically you cannot install rudder 5 on sles11

Reply:
I don't exactly know how migration from SLES 11 to SLES12 works, but I guess the old Postgresql could still be used (like on debian for example).